packages/contracts-rfq/test/FastBridgeV2.GasBench.Src.t.sol

```diff
@@ -50,13 +50,13 @@ contract FastBridgeV2GasBenchmarkSrcTest is FastBridgeV2SrcBaseTest {
 
     function createFixturesV2() public virtual override {
```

Comment:
This function is now basically a no-op after the parent's `createFixturesV2` is called (the default values for these structs residing in storage is zero). So you might as well remove the `createFixturesV2` definition in this particular test contract (no additional stuff is actually needed on top of the inherited fixture creation).